backend/routes.js

```diff
@@ -253,11 +297,29 @@ route.POST('/file', {
     // Finally, verify the size is correct
     if (ourSize !== manifest.size) return Boom.badRequest('Mismatched total size')
 
+    const manifestHash = createCID(manifestMeta.payload)
+
+    // Ensure that the manifest doesn't exist
+    if (await sbp('chelonia/db/get', manifestHash)) {
+      throw new Error(`Manifest ${manifestHash} already exists`)
+    }
+    // Ensure that the chunks do not exist
+    await Promise.all(chunks.map(async ([cid]) => {
+      const exists = !!(await sbp('chelonia/db/get', cid))
+      if (exists) {
+        throw new Error(`Chunk ${cid} already exists`)
+      }
+    }))
```

Comment:
It's not clear to me why we're making sure these chunks and everything else don't exist... if they do exist, is that a big problem? If so, could you add a comment explaining why?